workshops/fabric-e2e-rag/workshop.md

```diff
@@ -0,0 +1,606 @@
+
+## You'll learn how to: 
+
+- Pre-Process PDF Documents using [Azure AI Document Intelligence](https://azure.microsoft.com/products/ai-services/ai-document-intelligence) in Azure AI Services.
+- Perform text chunking using SynapseML.
+- Generate embeddings for the chunks using SynapseML and  [Azure OpenAI Services](https://azure.microsoft.com/products/cognitive-services/openai-service).
+- Store the embeddings in a vector store using [Azure Cognitive Search](https://azure.microsoft.com/products/search).
+- Build a question answering pipeline.
+
+
+## Pre-requisites
+
+
+| | |
+|----------------------|------------------------------------------------------|
+| Azure account        | [Get a free Azure account](https://azure.microsoft.com/free) |
+| Microsoft Fabric License | [Microsoft Fabric Licenses](https://learn.microsoft.com/en-us/fabric/enterprise/licenses?WT.mc_id=data-114676-jndemenge) |
+| A workspace in Microsoft Fabric | [Create a Microsoft Fabric workspace](https://learn.microsoft.com/en-gb/fabric/data-warehouse/tutorial-create-workspace?WT.mc_id=data-114676-jndemenge) |
+| Access to Azure OpenAI API | [Request access to Azure OpenAI](https://aka.ms/oaiapply) |
+| A Web browser        | [Get Microsoft Edge](https://www.microsoft.com/edge) |
+| Python knowledge | [Python for beginners](https://learn.microsoft.com/en-us/training/paths/beginner-python/) |
+
+
+---
+
+
+# Introduction
+
+Analyzing structured data has been an easy process for some time but the same cannot be said for unstructured data. Unstructured data, such as text, images, and videos, is more difficult to analyze and interpret. However, with the advent of advanced AI models, such as OpenAI's GPT-3 and GPT-4, it is now becoming easier to analyze and gain insights from unstructured data.
+
+An example of such analysis is the ability to query a document for specific information using natural language which is achievable though a combination of information retrieval and language generation.
+
+By leveraging the RAG (Retrieval-Augmented Generation) framework, you can create a powerful question-and-answering pipeline that uses a large language model (LLM) and you own data to generate responses. 
+
+The architecture of such an application is as shown below:
+
+![schema](assets/schema.png)
+
+To get an in-depth understanding of the RAG framework, refer to [this workshop](https://moaw.dev/workshop/gh:azure-samples/azure-openai-rag-workshop/base/docs/)
+
+
+---
+
+# Environment Setup
+To continue with this workshop, you'll need to create a Lakehouse in your Microsoft Fabric workspace and deploy the necessary resources in your Azure account. We'll detail the steps to do this below.
+
+## Create a Lakehouse
+To create a new Lakehouse in your Microsoft Fabric workspace, open the `Data Engineering workload` on the bottom left of the workspace and click on `Lakehouse`. Provide the name `rag_workshop` and click `Create`
```

Comment:
I cant find "Data Engineering workload", but I was able to find a "New" button and I found Lakehouse when I selected "More options" on that.

Reply:
![Screenshot 2024-02-14 at 12 53 08 PM](https://github.com/microsoft/moaw/assets/297042/8f77c181-a4c6-4c47-80bf-e293d5fde5ec)
![Screenshot 2024-02-14 at 12 52 57 PM](https://github.com/microsoft/moaw/assets/297042/d8cfdec1-078c-4850-b109-150dab9926e5)